packages/react-instantsearch-hooks/CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [6.39.0](https://github.com/algolia/instantsearch.js/compare/react-instantsearch-hooks@6.38.3...react-instantsearch-hooks@6.39.0) (2023-01-17)
+
+
+### Features
+
+* **react-instantsearch-hooks:** Add useStats hook ([#5425](https://github.com/algolia/instantsearch.js/issues/5425)) ([772c918](https://github.com/algolia/instantsearch.js/commit/772c918f47aec183af3f1aa78c65505f70dd0088))
```

Comment:
this should be copied in the other React InstantSearch Hooks based flavours, but because it's the only change I'm more inclined to skip the version. What do you think @FabienMotte ?

Reply:
Is this change required by another team?

Reply:
No, it was made by @cdhawke, but I believe he already has a workaround with custom hook for this.

Reply:
Makes sense to skip then 👍